### ISSUE TITLE
Form fields are missing labels for accessibility

### ISSUE THREAD

**Issue by davidbenque** [xwiki-labs/cryptpad#2195]
We have received reports that some fields in forms are missing proper labels which makes them confusing to screen reader users. We should ensure that all fields in forms are properly labelled.

**Comment by zuzanna-maria** [comment]
Completed with #2260 as of 2026.5
